packages/perennial/contracts/incentivizer/Incentivizer.sol

```diff
@@ -37,14 +37,15 @@ contract Incentivizer is IIncentivizer, UInitializable, UControllerProvider, URe
      * @param programInfo Parameters for the new program
      * @return programId New program's ID
      */
-    function create(IProduct product, ProgramInfo calldata programInfo)
+    function create(IProduct product, ProgramInfo memory programInfo)
     external
     nonReentrant
     isProduct(product)
     notPaused
     onlyOwner(programInfo.coordinatorId)
     returns (uint256 programId) {
         IController _controller = controller();
+        UFixed18 programTotal = programInfo.amount.sum();
```

Comment:
nit: I'd put this in the "Charge creator" section for clarity since it's not used elsewhere, or just remove / revert back to original

Reply:
This needs to be before the `deductFee` call because that modifies the programInfo to be a smaller total amount

Reply:
oh good point, maybe put it right above the deductFee call to make that more obvious?